src/main/java/uk/gov/hmcts/ccd/domain/model/aggregated/CaseViewFieldBuilder.java

```diff
@@ -1,21 +1,34 @@
 package uk.gov.hmcts.ccd.domain.model.aggregated;
```

Comment:
we said this code is no longer needed. Is that right? Can we remove it then?

Reply:
reverted